WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationDetailsViewController.swift

```diff
@@ -662,58 +657,6 @@ private extension NotificationDetailsViewController {
         cell.downloadGravatarWithURL(mediaURL)
     }
 
-    func setupActionsCell(_ cell: NoteBlockActionsTableViewCell, blockGroup: FormattableContentGroup) {
-        guard let commentBlock: FormattableCommentContent = blockGroup.blockOfKind(.comment) else {
```

Comment:
This check – `blockOfKind(.comment)` – is a guarantee that this content is never displayed by this screen.